src/integration/spring/src/jvmTest/kotlin/community/flock/wirespec/integration/spring/kotlin/generated/GetOrderByIdEndpoint.kt

```diff
@@ -0,0 +1,115 @@
+package community.flock.wirespec.integration.spring.kotlin.generated
```

Comment:
Why do we need all these generated files in source control?

Reply:
We could generate them with with gradle. It is only not possible to use the Wirespec gradle plugin for this.

Reply:
Ah ok